### DISCUSSION TITLE
RU update

### DISCUSSION THREAD

**Discussion by zuqzwanq** [aclap-dev/video-downloadhelper#966] [Language & Translation]
[ru.txt](https://github.com/user-attachments/files/16096145/ru.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales

**Comment by jettamb** [comment]
Here is my version
[ru.txt](https://github.com/user-attachments/files/16345419/ru.txt)
